code/modules/hydroponics/hydroponics.dm

```diff
@@ -1042,4 +1042,7 @@
 	else
 		..()
 
+/obj/machinery/hydroponics/soil/CanPass()
```

Comment:
To be honest, the parent `CanPass` should probably just be tweaked; the only purpose is to allow nymphs to move over top of the trays; you could just do a check for them, and if the `mover` is a nymph, it returns true--otherwise, *its* parent governs everything; then you're not having to have an override for this, yet again.